### PR TITLE
minor cleanup reflecting current procedures

### PR DESCRIPTION
**Pull request by zburke**

### CODE REVIEW

File: doc/release-procedure.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Ensure that each of the Jira issues listed in the change-log is tagged to the nu
 
 ### Check dependencies
 
-Make sure your `package.json` does not contain any unreleased dependencies -- for example, a bugfix version of stripes-core that adds a new facility, available via the CI repository `folioci` but not from an actual release. To check for this, make a brand new checkout of the module you're working on, outside of any Yarn workspace, switch back to the `npm-folio` registry, and try to install.
+Make sure your `package.json` does not contain any unreleased dependencies -- for example, a bugfix version of stripes that adds a new facility, available via the CI repository `folioci` but not from an actual release. To check for this, make a brand new checkout of the module you're working on, outside of any Yarn workspace, switch back to the `npm-folio` registry, and try to install.
```
Comment (MikeTaylor) [inline doc/release-procedure.md]: Maybe use code-style for `stripes` so it's clear that you're referring to the software package of that name, not to Stripes in a more general sense?

File: doc/release-procedure.md [hunk 2]
```diff
@@ -89,21 +88,14 @@ When breaking changes have been introduced, but are not stable enough to officia
 
 For example, given a current release of `1.2.3` in which breaking changes must be introduced, release these changes as `2.0.0-pre.1`.  If necessary, repeat with `2.0.0-pre.2` and so on until code is deemed stable enough to release `2.0.0`.
 
-Update dependant packages when needed to allow CI to pick up the pre-release version for integration testing.  A pre-release should be used in place of alternative references such as `latest` or a specific commit hash.  Once the final version has been released, any packages depending on the pre-release version should be updated.
-
-
-
-## Notes on dependencies
```
Comment (MikeTaylor) [inline doc/release-procedure.md]: Why is this section no longer relevant? It seems relevant to me, except that `stripes-core` should become `stripes`.

**Comment by MikeTaylor** [comment]
What is the deal with this year-old PR?

**Comment by zburke** [comment]
I just discovered this year-old PR too, @MikeTaylor, and added your helpful edits.

**Comment by MikeTaylor** [comment]
All right, let's merge this sucker, before the reference to "current procedures" gets more out of date! :-)

**Pull request merged**